packages/react-environment/src/api-providers/web3/web3-bridge-deposit.ts

```diff
@@ -210,28 +210,33 @@ export class Web3BridgeDeposit extends BridgeDeposit<WebbWeb3Provider, DepositPa
     return [];
   }
 
-  /*
-   *
-   *  Mixer id => the fixed deposit amount
-   * destChainId => the Chain the token will be bridged to
-   * If the wrappableAssetAddress is not provided, it is assumed to be the address of the webbToken
-   * */
+  /**
+   * Generates a bridge note for the given mixer and target destination chain.
+   * Note: If the wrappableAssetAddress is not provided, it is assumed to be
+   *       the address of the webbToken
+   * Note: This functione expects `destChainId` is EXPLICITLY the correctly computed
+   *       target chain id with the type encoded in its value.
+   * @param mixerId - the mixerId
+   * @param destChainId - encoded destination chain Id and chain type
+   * @param wrappableAssetAddress - the address of the token to wrap into the bridge
+   * @returns
+   */
   async generateBridgeNote(
     mixerId: number | string,
-    destChainId: ChainId,
+    destChainId: number,
     wrappableAssetAddress?: string
   ): Promise<DepositPayload> {
     const bridge = this.activeBridge;
     if (!bridge) {
       throw new Error('api not ready');
     }
     const tokenSymbol = bridge.currency.view.symbol;
-    const destEvmId = chainIdIntoEVMId(destChainId);
+    // const destEvmId = internalChainIdIntoEVMId(destChainId);
     const sourceEvmId = await this.inner.getChainId();
-    const deposit = createAnchor2Deposit(destEvmId);
+    const deposit = createAnchor2Deposit(destChainId);
```

Comment:
If we are to change this field directly, it will break existing notes.

Would it be better to change the jsnote structure to have a targetChainType field? @AhmedKorim

Reply:
This is a good time to bump our note version. We should then have a handler that handles legacy `v1` notes. WDYT @nepoche @AhmedKorim